orchagent/neighorch.cpp

```diff
@@ -1133,8 +1133,30 @@ void NeighOrch::doVoqSystemNeighTask(Consumer &consumer)
             }
 
             if (m_syncdNeighbors.find(neighbor_entry) == m_syncdNeighbors.end() ||
-                    m_syncdNeighbors[neighbor_entry].mac != mac_address)
+                    m_syncdNeighbors[neighbor_entry].mac != mac_address ||
```

Comment:
Why do we check mac for remote neighbor? Isn't checking encap sufficient?

Reply:
No. For mac change, the encap index is not changed. If we only check encap index, we'll not be processing the mac change.